carlymclift/RancidTomatillos
# Jkw movie page

#### What's this PR do?
- Refactors MoviePage to fetch data 
- Adds MoviePage styling
- Adds functionality to App to click on a movie card & see it's page
- Adds simple test file 
#### Where should the reviewer start?
- Check out the `MovieDetails` folder, then `App.js`
#### How should this be manually tested?
- Fetch remote branch, run `npm start`- make sure data loads & you can click on movie cards
#### What are the relevant tickets?
- Close #4 
#### Screenshots (if appropriate)
![Screen Shot 2020-08-22 at 11 52 33 AM](https://user-images.githubusercontent.com/62263439/90962524-fe76fb00-e46d-11ea-944b-0d80ed7c169a.png)
